src/app/static/src/js/components/reports/runReportInline.vue

```diff
@@ -18,7 +18,7 @@
         <button v-on:click="confirmRun" class="btn mt-2" type="submit">Run report</button>
```

Comment:
Out of curiosity, can we not use simply `:click` instead of `v-on:click`? Or do we not use it because it's not supported by all browsers?

Reply:
Do you mean `@click`? No reason at all, I do prefer the `@` syntax generally. I wonder if that line was written quite early on when we were less familiar with vue. I'll change it for consistency :)

Reply:
Yes sorry, I meant `@click`. I was getting it mixed up with `v-bind:`.